scripts/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# This script builds the networkservicemesh
+#
+
```

Comment:
I think we want to retain:

test -z $(go fmt ./...) || (echo "Run go fmt ./... and recommit your code";exit 1)

To do the format checking.

Reply:
I had thought I left that in, but I will add it back, as it does appear to be gone.

Reply:
Actually, the format check remains in the Dockerfile, see line 5 above. Let me know if you'd prefer it moved here.

Reply:
Ah... I missed that.  I could see either way, so long as we are deliberate about it :)

Reply:
If that's the case, let me move it, as it may make more sense to have it in one place.